ts/features/euCovidCert/screens/__test__/EUCovidCertValidScreen.test.tsx

```diff
@@ -84,22 +84,40 @@ describe("Test EUCovidCertificateValidScreen", () => {
     if (detailsButton) {
       fireEvent.press(detailsButton);
       const detailsPayload: NavigationAction = {
-        type: "Navigation/NAVIGATE",
-        routeName: "EUCOVIDCERT_MARKDOWN_DETAILS",
-        params: { markdownDetails: "markdownDetails" }
+        type: "NAVIGATE",
+        payload: {
+          name: "MESSAGES_NAVIGATOR",
```

Comment:
You're right, fixed in e4268782704b344d2194956ed782c8b9cae61707.
I'm going to replace all the strings with the consts! 
Btw, in the case of usage when we have the annotation with the navigation props, I think could be acceptable using directly the string, because in that case we have the typecheck and also the help of the IDE that suggests us only the available options, what you think?

Reply:
Yes 10/10 agree, I tried to add a comment only where we had hardcoded strings without typecheck (mostly in the tests). If in some comment I made there were the typecheck feel free to ignore it! 🙂